frontend/src/components/RisNavbar.vue

```diff
@@ -1,11 +1,28 @@
 <script setup lang="ts">
 import neurisLogo from "@/assets/neuRIS-logo.svg"
 import { useAuthentication } from "@/lib/auth"
-import { RouterLink } from "vue-router"
+import { RouterLink, useRoute } from "vue-router"
 import UseOutline from "~icons/ic/baseline-person-outline"
+import { computed } from "vue"
 
 const { getUsername, getLogoutLink } = useAuthentication()
 const logoutLink = getLogoutLink()
+const route = useRoute()
+
+const activeTab = computed(() => {
+  if (route.path.startsWith("/verkuendungen")) {
+    return "verkuendungen"
+  } else if (route.path.startsWith("/bestand")) {
+    return "bestand"
+  }
+
+  return "verkuendungen"
+})
+
+const tabs = [
+  { id: "verkuendungen", label: "Verkündungen" },
+  { id: "bestand", label: "Bestand" },
```

Comment:
how about `route` property in here for every tab? that way we would only need to update one spot if we have a need to add another route (or change its path) and not touch 3 places (and might miss one).

Reply:
good suggestion thanks!